src/parser.js

```diff
@@ -54,4 +54,24 @@ function parseWithBabylon(text) {
   });
 }
 
-module.exports = { parseWithFlow, parseWithBabylon };
+function parseWithTypeScript(text) {
+    const parser = require('typescript-eslint-parser')
+    const result = parser.parse(text, {
+        loc: true,
+        range: true,
+        tokens: true,
+        attachComment: true,
+        ecmaFeatures: {
+            jsx: true,
+        }
+    })
+    return JSON.parse(JSON.stringify(result, function(key, value) {
+        if ((key === "start" || key === "end") && typeof value === "number") {
+            return undefined;
```

Comment:
We don't 😄 sorry we need to do this for our tests for the parser, it's a hang up from espree being a fork of acorn. Removing!